progressbar/widgets.py

```diff
@@ -260,7 +260,8 @@ def __init__(
     def _calculate_eta(self, progress, data, value, elapsed):
         '''Updates the widget to show the ETA or total time when finished.'''
         if elapsed and value:
-            eta_seconds = elapsed * progress.max_value / value - elapsed
```

Comment:
Can you explain why you are using `data['value']` instead of `value`?

If everything works as expected, `value` should either be identical to `data['value']` or be the sampled version of `value`.